pcap.c

```diff
@@ -4211,6 +4211,14 @@ pcap_read_dead(pcap_t *p, int cnt _U_, pcap_handler callback _U_,
 	return (-1);
 }
 
+static int
```

Comment:
A `breakloop_op` has type `void (*)(pcap_t *)`, not `int (*)(pcap_t *)`, so this should have that type.

Reply:
Oh my bad